### PR TITLE
Deprecation of mne-realtime

### PR DESCRIPTION
**Pull request by mscheltienne**
Add warning to readme and doc/index redirecting to `mne-lsl`. 
Closes https://github.com/mne-tools/mne-lsl/issues/159

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -3,7 +3,83 @@
...
 
-.. include:: ../README.rst
+MNE-realtime
+============
+
+.. warning::
+
+    This project is discontinued in favor of [MNE-LSL](https://github.com/mne-tools/mne-lsl).
+    At the moment, [MNE-LSL](https://github.com/mne-tools/mne-lsl) replaces the ``LSLClient``
+    and does not yet support the FieldTrip buffer.
```
Comment (mscheltienne) [inline doc/index.rst:13]:
```suggestion
    This project is discontinued in favor of https://github.com/mne-tools/mne-lsl.
    At the moment, https://github.com/mne-tools/mne-lsl replaces the ``LSLClient``
    and does not yet support the FieldTrip buffer.
```

**Comment by mscheltienne** [comment]
@larsoner Happy with the deprecation messages?

File: setup.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 DISTNAME = 'mne-realtime'
 DESCRIPTION = 'A module for real-time data analysis with MNE.'
-with codecs.open('README.rst', encoding='utf-8-sig') as f:
+with codecs.open('README.md', encoding='utf-8-sig') as f:
```
Comment (larsoner) [inline setup.py:16]: You should also change the content type somewhere to say it's markdown. It might autodetect it, not sure, but better to be explicit.

**Review by larsoner — COMMENTED** [review]
This is good but to really drive the point home we should also `warnings.warn(..., FutureWarning, stacklevel=2)` or so in `mne_realtime/__init__.py`.

File: setup.py (hunk 2 above)
Comment (larsoner) [reply setup.py:16]: `long_description_content_type="text/markdown"` maybe?

**Comment by mscheltienne** [comment]
That should cover it. Should we do a release of `mne-realtime` just to have the warning in?

**Comment by larsoner** [comment]
https://pypi.org/project/mne-realtime/0.3.0/

**Pull request merged**